src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -83,9 +88,22 @@ void RequestFocus()
 					return;
 
 				platformView?.RequestFocus();
```

Comment:
this returns a bool based on if this worked or not. I think it'd be useful to check the result here. 

I tested it
```C#
				if (platformView?.RequestFocus() == true)
					focusRequested?.Invoke();
```

and it seems to work fine.